### DISCUSSION TITLE
vscode 安装UnoCSS扩展后，编辑器卡顿

### DISCUSSION THREAD

**Discussion by PdxLook** [unocss/unocss#3800] [Q&A]
vscode 安装UnoCSS扩展后，编辑器卡顿；禁用此插件后就正常了

**Comment by zhl1232** [comment]
![img_v3_02g5_7f41ea0a-a0e9-4e8b-ad71-f46e776561fg](https://github.com/user-attachments/assets/abff18d5-7b0f-4acd-b25f-076557ecfd12)

**Comment by zhl1232** [comment]
[antfu.unocss-unresponsive.cpuprofile.txt](https://github.com/user-attachments/files/17568296/antfu.unocss-unresponsive.cpuprofile.txt)
这个是提示上传的文件，不确定有没有帮助
